java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1011,4 +1011,21 @@ public void zcard(BaseClient client) {
                 assertThrows(ExecutionException.class, () -> client.zcard("foo").get());
         assertTrue(executionException.getCause() instanceof RequestException);
     }
+
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void pfadd(BaseClient client) {
+        String key = UUID.randomUUID().toString();
+        assertEquals(1, client.pfadd(key, new String[0]).get());
+        assertEquals(1, client.pfadd(key, new String[] {"one", "two"}).get());
+        assertEquals(0, client.pfadd(key, new String[] {"two"}).get());
+        assertEquals(0, client.pfadd(key, new String[0]).get());
```

Comment:
This is a no-op?  I don't think there's much value to this last step.

Reply:
Yes, it is a no-op. Shows the diff vs the first test (line 1020).